src/lang/zh-TW.json

```
@@ -126,7 +126,7 @@
   "label.reset": "重設",
   "label.reset-website": "重設網站",
   "label.retention": "保留",
-  "label.retention-description": "Measure your website stickiness by tracking how often users return.",
+  "label.retention-description": "通過跟踪用戶返回的頻率來衡量網站的用戶粘性",
```

With all due respect, but some expressions in the translation may not align with the Traditional Chinese usage in Taiwan.

```suggestion
  "label.retention-description": "透過追蹤使用者回訪的頻率來衡量您的網站黏著度。",
```

---

This is correct, it has been fixed